CodeSniffer/Standards/Generic/Sniffs/WhiteSpace/DisallowSpaceIndentSniff.php

```diff
@@ -94,42 +94,60 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
                 continue;
             }
 
-            // If tabs are being converted to spaces, the original content
+            // If spaces are being converted to tabs, the original content
```

Comment:
This comment is referring to PHPCS converting tabs to spaces during tokenizing, so the original comment is correct in this case.